lib_parts.ks

```diff
@@ -1,190 +1,219 @@
-FUNCTION partsDoIt {
```

Comment:
I am used to ts=4, but have no problems with two spaces. Both have their advantages.

Reply:
If you write up a PR for a root-level file named `STYLESHEET.md` with your thoughts, we can nail down the details together and ask for the owner's blessing.  If it's approved, we can use it moving forward for new code and eventually update the code base.

My goals for a stylesheet PR are to ensure the code is consistent and easy to read/understand *and* to conserve characters.  Using tabs instead of spaces is the smart move, and I apologize for not thinking about it originally.